openshift/matomo/openshift/templates/mariadb/mariadb-build.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
does this file mean we're going to have a MariaDB server running as well as a Postgres server?

Reply:
Yep. Only works with MySQL and MariaDB : https://matomo.org/faq/how-to-install/faq_55/